docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,78 @@
 # Setting up PyCharm with Evennia
 
-[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
+[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. 
+It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
 
-> This page was originally tested on Windows (so use Windows-style path examples), but should work the same for all platforms.
+First, download and install the IDE edition of your choosing.
+The professional edition has integrated support for Django which can
+make certain steps easier but the community edition has everything you need.
 
-First, install Evennia on your local machine with [[Getting Started]]. If you're new to PyCharm, loading your project is as easy as selecting the `Open` option when PyCharm starts, and browsing to your game folder (the one created with `evennia --init`). We refer to it as `mygame` here. 
+After setting it up, you want to create a new project.
+1. Click on the new project button.
+2. Select the location for your project.
+You should create two new folders, one for the root of your project and one
```

Comment:
These instructions will not get you a functional game. The separate 'game folder' will not have been initialized to work as an Evennia game dir.

Reply:
Right, its tricky because the IDE works best when opening the folder directly and it stores its configuration folder there too, I will revisit this step.